hack/prerelease.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null
+    pwd
+)"
+
+PROJECT_DIR="$(
+    cd "$SCRIPT_DIR/.." >/dev/null
+    pwd
+)"
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Optional arguments:
+    -p, --product $PRODUCT
```

Comment:
I think you shouldn't use a variable `$PRODUCT`,  it will fail with error `PRODUCT: unbound variable`

Reply:
Done